conference.js

```diff
@@ -601,6 +603,7 @@ export default {
                     return [];
                 });
         } else if (requestedAudio || requestedVideo) {
+            APP.store.dispatch(gumPending(initialDevices, IGUMPendingState.PENDING_UNMUTE));
```

Comment:
Not really. The thing is that we will need to add extra param to `createLocalTracksF` in order to indicate we want to do the whole pending thing. And even if we do it we will still have 2 out of 5 places (for the track.unmute and `createLocalTracksA` use cases ) where the pending logic will be done separately. So I'm not sure it will bring any value...

But I'm not against it so if you like it better I can move it in `createLocalTracksF` and use a new parameter to check if we need to execute the logic or not. Do you want me to do it?

Reply:
Yes, I think we should go ahead and do it so that its more clear. We can optimize the gUM pending logic in the future but having it all in `createLocalTracksF` will definitely help.